.docker/images/govcms7/scripts/govcms-deploy

```diff
@@ -40,13 +53,15 @@ if [[ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
 # Production environments.
 else
 
-  if tables=$(drush sqlq 'show tables;') && [ -n "$tables" ]; then
+    query_opts=[ -z "$MARIADB_HOST_READREPLICA" ] "--database=read" || ""
```

Comment:
Where is the `MARIADB_HOST_READREPLICA` value actually getting set?

Reply:
Replaced with `MARIADB_READREPLICA_HOST` -- expected env var provided via Lagoon